### DISCUSSION TITLE
JWT Support

### DISCUSSION THREAD

**Discussion by dialtonetech** [dotnet/WatsonWebserver#71] [Q&A]
Does the server accept JWT for authentication or is their another process available for security?

Is it also possible to enable swagger for the end points?

**Comment by jchristn** [comment]
Hi, Watson itself doesn't implement any form of authentication. Authentication is domain specific (ie knowledge of your users and their credentials) so it isn't part of the library. 

Watson doesn't natively provide an OpenAPI JSON today. 

Cheers, Joel
